src/screens/GroupSetting/index.js

```diff
@@ -103,7 +105,7 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[600],
     fontSize: 14,
     lineHeight: 16.94,
-    color: COLORS.holytosca,
+    color: colors.darkBlue,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed from `COLORS.holytosca` to `colors.darkBlue`. If this is intentional, ensure that `colors.darkBlue` is defined in your colors utility module.


<!-- This is an auto-generated comment by OSS CodeRabbit -->